src/generic_ui/locales/en/messages.json

```diff
@@ -667,6 +667,14 @@
     "description": "Notification for when the user has been logged out of the social network.",
     "message": "You have been logged out of __network__"
   },
+  "PRE_LOG_OUT": {
+  "description": "Notification to ask user if they are sure that they want to log out.",
+  "message": "Are you sure you want to log out? You are granting access to __name__ and __numOthers__ others"
+  },
+  "PRE_LOG_OUT_SIMPLE": {
+    "description": "Notification to ask user if they are sure that they want to log out.",
+    "message": "Are you sure you want to log out?"
```

Comment:
It looks like you are using this message if the user is sharing access with only one person. It'd be great to still show the user's name, something like
"Are you sure you want to log out? You are granting access to __name__."